Update the-ide-git-button.md (#1118)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md

</p>

**Pull changes from master/main**: This option will be available if you are on any local branch that is behind the remote version of the base branch or the remote version of the branch that you're currently on.

## Reclone Repo

> Reclone your repository directly from the dbt IDE and your browser. You can reset your repository back to a fresh clone from your remote, you will want to click on the bottom right hand side green 'Ready'text, then click on the 'Reclone Repo' selection.

Reviewer: These changes bring over changes from [this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/1117). 

Note that in the future we'll be able to write this in one place and refer to it form both of these places. Here's the [PR for tracking this](https://github.com/dbt-labs/docs.getdbt.com/issues/1126).
```suggestion
Reclone your repository directly from the dbt IDE and your browser. To reset your repository back to a fresh clone from your remote, click **Ready**, which is green text in the bottom-right, then click **Reclone Repo**.
```

Author: hi @runleonarun , i've added the updated video today via the terminal today so it should no longer be tiny (hopefully) - please let me know if anything else is needed!

terminal confirmation:
```
Your branch is up to date with 'origin/mirnawong1-patch-17'.

nothing to commit, working tree clean
mirnawong@mirna-wong dbt-cloud % 
```
